Netkan/Program.cs

```diff
@@ -30,21 +31,59 @@ public static class Program
 
         public static int Main(string[] args)
         {
+            if (args.Any(i => i == "--debugger"))
+            {
+                Debugger.Launch();
+            }
+
+            var parser = new Parser(c => c.HelpWriter = null).ParseArguments<CmdLineOptions>(args);
+            parser.WithParsed(opt => Run(opt));
+            parser.WithNotParsed(errs =>
+            {
+                if (errs.IsVersion())
+                {
+                    Console.WriteLine(Meta.GetVersion(VersionFormat.Full));
+                }
+                else
+                {
+                    HelpText ht = HelpText.AutoBuild(parser, h =>
+                    {
+                        h.AddDashesToOption = true;
+                        h.AutoHelp = false;
+                        h.Heading = $"NetKAN {Meta.GetVersion(VersionFormat.Full)}";
+                        h.Copyright = $"Copyright � 2014-{DateTime.Now.Year}";
```

Comment:
```suggestion
                        h.Copyright = $"Copyright © 2014-{DateTime.Now.Year}";
```

I think that's not a copyright symbol.